src/ui/index.js

```diff
@@ -8,6 +8,13 @@ import UiExports from './ui_exports';
 import UiBundle from './ui_bundle';
 import UiBundleCollection from './ui_bundle_collection';
 import UiBundlerEnv from './ui_bundler_env';
+import i18nPlugin from '../plugins/i18n/server/i18n/index';
+import langParser from 'accept-language-parser';
+
+let kibanaTranslations = [];
+let acceptLanguages = '';
+
+const DEFAULT_LANGUAGE = 'en';
```

Comment:
* good about fallback. It might be worth documenting this further - something like, "source content is assumed to be in English".

* also, for a particular Kibana instance, they could - conceivably- have a different default language.